wiki/Disambiguation/de.md

```diff
@@ -0,0 +1,10 @@
+# Begriffserklärung
+
+Die Begriffserklärung ist ein Prozess der Lösung von Konflikten die entstehen können, wenn der Titel eines Artikels nicht eindeutig ist, meistens da er sich auf mehr als nur ein vom osu! wiki behandeltes Thema bezieht, entweder als Haupt- oder Unterthema.
```

Comment:
Das ist wirklich schwer zu lesen. Vielleicht in 2 Sätze aufteilen oder umformulieren.